### PR TITLE
Initial commit of Euler 1

### PR DESCRIPTION
**Pull request by gparsloe**
Used the Brackets editor with jslint.

### CODE REVIEW

**Comment by gparsloe** [comment]
Oops. Just realized I need to go to 1000. Hold on...

File: 1GP.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*jslint plusplus: true */
+/*global console*/
+
+/* https://projecteuler.net/problem=1
+
+If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
```
Comment (gotrevor) [inline 1GP.js]: This line is too long.  Limit to 120 characters or so per line?

File: 1GP.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Find the sum of all the multiples of 3 or 5 below 1000.
+*/
+
+(function () {
+    'use strict';
+    var counter = 0,
+        accumulator = 0;
```
Comment (gotrevor) [inline 1GP.js]: I prefer one `var` declaration per line.  I'd also prefer these to be `let` or `const`.
http://eslint.org/docs/rules/one-var

File: 1GP.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    for (counter = 1; counter < 1000; counter++) {
+        console.log("hey");
```
Comment (gotrevor) [inline 1GP.js]: That's a lot of `hey`!

**Comment by gotrevor** [comment]
Looks good!  Just a few minor comments.  Also, I prefer `eslint` or `jshint`.  But, we can tackle that later.

**Comment by gotrevor** [comment]
You did a lot of work to get here!  I'm impressed.  :)

**Comment by gparsloe** [comment]
Two questions:
1. Do we always need to delete a dev branch once it's merged? 
2. Should I be creating a new pull request every time I modify and commit a file?

**Comment by gotrevor** [comment]
1. I usually do one branch per PR, then delete the branch once it's merged.  I think of development branches as ephemeral.
2. Only need to put in a new PR if you have a new branch, or if you push changes to a branch who's PR has already been merged.

Also:
- respond to the comments in the PR in the PR.  I like that your commit messages address the PR comments - just go to the PR comment, and reply (just pasting the relevant commit hash is sufficient - github will even create an automatic link for something that looks like a commit hash).

**Comment by gparsloe** [comment]
Testing pasting commit hash: 7022b2fed30b8cfa3cbd6dffffb533c99a55674e

**Comment by gparsloe** [comment]
Hey, cool. It worked (got a link to the commit).

**Pull request merged**